src/main/steps/c100-rebuild/permission/content.ts

```diff
@@ -0,0 +1,94 @@
+import { YesOrNo } from '../../../app/case/definition';
+
+import { TranslationFn } from './../../../app/controller/GetController';
+import { FormContent } from './../../../app/form/Form';
+import { isFieldFilledIn } from './../../../app/form/validation';
+
+const en = {
```

Comment:
@ShakiraKP Can you please define it as function for en & cy constants ?
For example
`const en = () => ({
  title: 'Will you be using a legal representative in these proceedings?',
});`